dev/airflow-github

```diff
@@ -335,18 +333,28 @@ def changelog(previous_version, target_version, github_token):
             if match:
                 existing_commits.add(match.group(1))
 
-    for commit in log:
-        tickets = pr_title_re.findall(commit["subject"])
-        if tickets:
-            issue = gh_repo.get_issue(number=int(tickets[0][1:]))
-            issue_type = get_issue_type(issue)
-            files = files_touched(repo, commit["id"])
-            if commit["id"] in existing_commits:
-                continue
-            if is_core_commit(files):
-                sections[issue_type].append(commit["subject"])
-        else:
-            sections[DEFAULT_SECTION_NAME].append(commit["subject"])
+    # To disable the progress bar, set to True
+    disable_progress = False
```

Comment:
Should we have this as an option e.g --disable-progress?

Reply:
Yeah seems reasonable then having to change the code, let me make that change